testdata/project-v3-addon/go.mod

```diff
@@ -9,5 +9,5 @@ require (
 	k8s.io/apimachinery v0.19.2
 	k8s.io/client-go v0.19.2
 	sigs.k8s.io/controller-runtime v0.7.2
-	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20210113160450-b84d99da0217
+	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200522144838-848d48e5b073
```

Comment:
`declarative/v1` was checking if the plugin was `go/v2` or `go/v3` and using the appropiate SHA. The check seems to be broken. You will need to fix it.

Reply:
Do you know where? Because we did not change that at all.